flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/module/hive/HiveModule.java

```diff
@@ -81,6 +82,18 @@ public class HiveModule implements Module {
                                     "tumble_rowtime",
                                     "tumble_start")));
 
+    static final Set<String> FLINK_BUILT_IN_FUNC_FOR_HIVE =
```

Comment:
In [FLINK-30579](https://github.com/apache/flink/pull/21629) pr, we will introduce an option to enable native hive sum aggregate function, so I think this set should only contain the agg function name, and exclude other functions.

Reply:
I've address your comments.